frontend/RaffleBay/RaffleBay/View/SignupView.swift

```diff
@@ -0,0 +1,225 @@
+//
+//  SignupView.swift
+//  RaffleBay
+//
+//  Created by Meera Rachamallu on 2/6/20.
+//  Copyright © 2020 Meera Rachamallu. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+import SwiftyJSON
+
+//Button Gradient
+let ButtonGradient = LinearGradient(gradient: Gradient(colors: [Color("PurpleBlue"), Color("LightBlue")]), startPoint: .leading, endPoint: .trailing)
+
+//Size of the frames for the Text Fields - used to adjust keyboard hiding.
+var signupFrameHeight: CGFloat = 60
+
+//UI SignUp Style - DRY Principle
+struct SignUpStyle : TextFieldStyle {
+    func _body(configuration: TextField<Self._Label>) -> some View {
+        
+            VStack{
+            configuration
+                .font(.custom("Poppins", size: 24))
+                .padding()
+                .frame(height: signupFrameHeight)
+
+                Rectangle()
+                    .frame(height: 1.0, alignment: .bottom)
+                    .foregroundColor(Color("LightGray"))
+                    .offset(y:-10)
+            }
+    }
+}
+
+//TO DO:
+//Figure out a way when clicking outside of the form area
+//the keyboard should be dismissed.
+
+
+struct SignupView : View {
+    
+    //I don't know if this is the best/correct way to accept input.
+    //Another way I think could work is like a hashmap that maps inputs to their associated values. Will look into that.
+    @State private var firstName: String = ""
```

Comment:
add these variables to the user.swift file in the model file. then make a user object in this class and update it's variables